druid/src/window.rs

```diff
@@ -215,6 +226,17 @@ impl<T: Data> Window<T> {
 
         self.post_event_processing(queue, data, env, false);
 
+        //In some platforms, timer tokens are reused. So it is necessary to remove token from
+        //window's timer before adding base state's timers to it.
```

Comment:
*~In~ **On** some platforms, timer tokens are reused. So it is necessary to remove **the** token from **the** window's timer **map** before adding ~base state's timers~ **new tokens** to it.*